SU2_CFD/include/integration_structure.hpp

```diff
@@ -287,9 +287,9 @@ class CIntegration {
    * \param[in] RunTime_EqSystem - System of equations which is going to be solved.
    * \param[in] Iteration - Current iteration.
    */
-  virtual void MultiGrid_Cycle(CGeometry ***geometry, CSolver ****solver_container, CNumerics *****numerics_container,
+  virtual void MultiGrid_Cycle(CGeometry ****geometry, CSolver *****solver_container, CNumerics ******numerics_container,
                  CConfig **config, unsigned short iMesh, unsigned short mu, unsigned short RunTime_EqSystem,
-                 unsigned long Iteration, unsigned short iZone);
+                 unsigned long Iteration, unsigned short iZone, unsigned short iInst);
```

Comment:
I'm trying to avoid "feature specific" names. For now the only application for iInst is HB, but I wouldn't limit it to this.

Reply:
I would maybe then change things in the code you introduced like `nInst[iZone] = config_container[iZone]->GetnTimeInstances();` to be consitent.

I would personally prefer iInstance vs iInst, but it's just a personal preference :)

Reply:
Yes, I agree. I didn't change it before to avoid messing up (more than necessary) with the HB stuff.
I also like iInstance; however, the code is already very chunky, with extremely long lines. That's why I cut it to iInst. I would like to do the same with other variables: we have function calls that take 3 and 4 lines, just because of the length of the variables...